aws_lambda_fsm/handler.py

```diff
@@ -75,11 +101,23 @@ def lambda_api_handler(lambda_event):
         logger.exception('Critical error handling lambda: %s', lambda_event)
 
 
+def lambda_step_handler(lambda_event):
+    """
+    AWS Lambda handler for executing state machines.
+
+    :param lambda_event: a dict event from AWS Lambda
+    :return: a dict event to pass along to AWS Step Functions orchestration
+    """
+    obj = {OBJ.SOURCE: AWS.STEP_FUNCTION}
+    payload = json.dumps(lambda_event)  # Step Function just passes straight though
+    return _process_payload_step(payload, obj)
```

Comment:
It doesn't look like you're able to return Errors in your states, are you planning on adding that at some point?